.github/run_browserstack_acceptance.sh

```diff
@@ -11,7 +11,8 @@ if [[ $GITHUB_REF_NAME == release/*
   || $GITHUB_REF_NAME == master
   || $GITHUB_REF_NAME == support/* ]]
 then
-  npx testcafe "browserstack:ie@11.0,browserstack:safari" --config-file ./.github/testcafe.json -q
+  # npx testcafe "browserstack:ie@11.0,browserstack:safari" --config-file ./.github/testcafe.json -q
+  npx testcafe "browserstack:ie@11.0" --config-file ./.github/testcafe.json -q
```

Comment:
The issue with browserstack acceptance test hanging is coming from safari specifically. Talked to tom, he's okay with ignoring it for now. I removed it from this script so so the deploy actions would pass properly, will add it back after the release.